repos/system_upgrade/common/actors/initramfs/upgradeinitramfsgenerator/libraries/upgradeinitramfsgenerator.py

```diff
@@ -19,6 +19,7 @@
 
 INITRAM_GEN_SCRIPT_NAME = 'generate-initram.sh'
 DRACUT_DIR = '/dracut'
+DEDICATED_URL = 'https://access.redhat.com/solutions/7011704'
```

Comment:
The name `DEDICATED_URL` is not very helpful, I think something like `DEDICATED_LEAPP_PART_URL` would be better.

The same goes for `userspacegen.py`

Reply:
updated in both files.